zitadel/v2/human_user/resource.go

```diff
@@ -77,16 +77,16 @@ func GetResource() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "Preferred language of the user",
-				Computed:    true,
+				Default:     defaultPreferredLanguage,
```

Comment:
doesn't ZITADEL set this value itself if it gets an empty string?

Reply:
that's the problem, as in terraform an empty string can't be mapped and results in 0 which in turn gets mapped by ZITADEL to the default, so to handle it correctly in the state, the default is set if nothing is provided

Reply:
> results in 0 which in turn gets mapped by ZITADEL to the default,
isn't that exaxtly what we want? But ok, you know what you are doing.